front/app/containers/Admin/projects/project/analysis/InputPreview/components/FilterToggleButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { IconTooltip, Button, Box } from '@citizenlab/cl2-component-library';
+import { xor } from 'lodash-es';
+
+import { trackEventByName } from 'utils/analytics';
+import { useIntl } from 'utils/cl-intl';
+import { updateSearchParams } from 'utils/cl-router/updateSearchParams';
+
+import useAnalysisFilterParams from '../../hooks/useAnalysisFilterParams';
+import messages from '../../messages';
+import tracks from '../../tracks';
+
+export const FilterToggleButton = ({ customFieldId, value }) => {
+  const { formatMessage } = useIntl();
```

Comment:
No changes made to this component - just moved it into it's own file instead of being in the LongFieldValue file.